fitbenchmarking/fitbenchmarking
# Issue #323: Add default value for x in eval_f

#### Description of Work

Fixes #323 
Adds a default option and updates controllers that can make use of it


#### Testing Instructions

1. Check that answers are the same as master on a sample of the problems and minimizers

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
